rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -10,18 +10,19 @@
 from typing import Optional
 from typing import Text
 
-from rasa_nlu.components import Component
+from rasa_nlu.extractors import EntityExtractor
 from rasa_nlu.model import Metadata
 
 
 if typing.TYPE_CHECKING:
     from duckling import DucklingWrapper
 
 
-DUCKLING_PROCESSING_MODES = ["replace", "append"]
+DUCKLING_DIMENSIONS = ["time", "temperature", "number", "ordinal", "distance", "volume",
+                       "amount-of-money", "duration", "email", "url", "phone-number"]
```

Comment:
There is a list of available dimensions here, https://github.com/FraBle/python-duckling/blob/master/duckling/dim.py. I think we should use these constants. (and i think you missed timezone)